csrank/util.py

```diff
@@ -10,6 +10,14 @@
            'time_from_now', 'get_duration_seconds', 'setup_logging', 'progress_bar']
 
 
+def raise_missing_extra(package, extra):
+    """Notify the user about a missing package"""
+    raise Exception(
+        f'Could not import {package}.'
+        f'Please install it explicitly or specify the "{extra}" extra when installing this package.'
+    )
```

Comment:
Hm, did we gain anything by catching the `ImportError` and instead raising a generic `Exception`?

Reply:
Yes I think this is useful because of the extra context. The users will know *why* the dependency is missing and what they can do to fix it.

Reply:
That’s a good idea, but I would still raise a specific exception like `ImportError` or define a custom one like `MissingDependency`. That way users can catch it, if need be.

Reply:
Ah, you're absolutely right. I thought your point was about catching it at all, not about the generic `Exception`.

Do you think the approach is otherwise fine?